languages/js/src/helpers.ts

```diff
@@ -9,6 +9,7 @@ import {
   QueryEventKind,
 } from './types';
 import type { Class, obj, PolarTerm, QueryEvent } from './types';
+import isEqual = require('lodash.isequal');
```

Comment:
I don't think I've ever seen an `import ... = require(...)` statement — what's the difference between this and `import ... from ...`?

Reply:
I'd never seen it either ... according to error messages from node we would need to add an config option to use the normal syntax with `lodash.isequal` for some reason.

Reply:
Did you try `import * as isEqual from "lodash.isequal"`? I think that might be a more idiomatic way of doing this